src/components/sponsor-tiers/sponsor-tiers.tsx

```diff
@@ -72,7 +72,10 @@ const SponsorTier = ({
       : price;
 
   return (
-    <div className="bg-white text-black rounded-2xl p-6 relative not-prose">
+    <div
+      className="bg-white text-black rounded-2xl p-6 relative not-prose"
+      style={{ zIndex: 0 }}
```

Comment:
I think you can do `z-0` with tailwind, but what does it do in this case?

Reply:
Yep, z-0 is nicer. Without setting the z-index, the ribbon is sent all the way back:
![image](https://github.com/EuroPython/website/assets/75242929/0e3f15e7-fcf6-4456-a9fd-39cc5d9ca0e9)

And if the ribbon's z-index is anything other than ``-1``, it stays on top of the text.